osu.Game/Online/API/Requests/Responses/Comment.cs

```diff
@@ -40,11 +38,16 @@ public class Comment
         [JsonProperty(@"votes_count")]
         public int VotesCount { get; set; }
 
+        public CommentableType CommentableType;
+
         [JsonProperty(@"commenatble_type")]
-        public string CommentableType { get; set; }
+        private string commentableType
+        {
+            set => Enum.TryParse(value.Pascalize().Replace("_", string.Empty), true, out CommentableType);
```

Comment:
Pretty sure [`.Pascalize()` will already remove underscores](https://humanizr.net/#pascalize), so the `.Replace()` is gratuitous.